frontend/sw360-portlet/src/main/webapp/html/search/view.jsp

```diff
@@ -44,7 +44,7 @@
             <tbody style="background-color: #f8f7f7; border: none;">
             <tr>
                 <td>
-                    <input type="text" maxlength="100" style="width: 200px; padding: 5px; color: gray;height:20px;"
```

Comment:
I'm not sure why `maxlength="100"` was there, but is there a reason why you removed it?